src/service/randomNumberInRange.js

```diff
@@ -0,0 +1,44 @@
+// min should be a minimal number, max should be a maximal number, forbiddenNumbersArray should be an array containing all the numbers that should not be generated
+
+export const randomNumberInRange = (min, max, forbiddenNumbersArray = []) => {
+    
+    if (typeof min === 'number' && typeof max === 'number' && Array.isArray(forbiddenNumbersArray) && min <= max) {
+
+        // check if not every possible number is forbidden
+        if (max - min + 1 === forbiddenNumbersArray.length) {
+
+            const allPossibleOutComes = [];
+
+            for (let i = min; i <= max; i++) {
+                allPossibleOutComes.push(i);
+            }
+
+            const allPossibleOutComesSorted = allPossibleOutComes.sort((a, b) => {return a-b});
+            const forbiddenNumbersArraySorted = forbiddenNumbersArray.sort((a, b) => {return a-b});
+
+            if  (allPossibleOutComesSorted.every((val, index) => val === forbiddenNumbersArraySorted[index])) {
+                throw Error('Invalid arguments!')
```

Comment:
[Readability and error handling]
When you are designing error messages think about other people that will stumble upon it.
Ask yourself:
Does this error message tell some valuable informations why it happened?
Does this error show exactly what values are not ok?

Both of those questions helps the reader understands what is happening when he is using your code.